mlir/utils/jenkins/build-dockerfile.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+set -x
+
+declare -a ARGS=("$@")
+
+err() {
+  echo "$*" >&2
+}
+
+# Clean up the docker images
+# Note: includes a workaround for the occasional hung docker daemon
+start_clean_docker() {
+  systemctl status docker | grep 'Active:'
+  sudo /usr/bin/pkill -f docker
```

Comment:
@krzysz00 what was the intention behind this line ? was it to stop and remove the running containers ?